training/heterogeneous-clusters/pt.grpc.local/README.md

```diff
@@ -0,0 +1,106 @@
+### Local Training using gRPC Client-Server
+
+The example here provides a conceptual idea of how to separate data preprocessing and training processes, and establish a gRPC client server communication between these components. We re-use the same client-server communication implementation in our Heterocluster SageMaker training example <PUT THE LINK when we move to aws-samples>. To demonstrate this we first run baseline testing where both the components (data preprocessing and training) run in a set of processes (see section A). In Section B, we split data processing run on gRPC Server set of processes whereas training run on a different set of processes.
+
+Important:
```

Comment:
You should probably also have this information on an index.rst or in a notebook. As far as I can tell now, it's not testable, which can be an issue over time. How do you assure quality of this example? Can you put these commands and info in a notebook, so the flow can be tested?